File: frontend/ReactMusicCommerce/src/responsemodel/CheckoutResponse.ts
export interface CheckoutResponse {
  orderId: number;
  paymentStatus: string;
  paymentMethod: string;
  gatewayTransactionCode?: string | null;
  totalItems: number;
  totalAmount: number;
  createdAt: string;
  paymentUrl?: string | null;
  message: string;
}

File: backend/music/src/main/java/com/springboot/music/responsemodel/CheckoutResponse.java
package com.springboot.music.responsemodel;

import lombok.*;

import java.time.LocalDateTime;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class CheckoutResponse {

    private Integer orderId;
    private String paymentStatus;
    private String paymentMethod;
    private String gatewayTransactionCode;
    private Integer totalItems;
    private Double totalAmount;
    private LocalDateTime createdAt;
    private String paymentUrl;
    private String message;
}

